src/build.d

```diff
@@ -1562,3 +1606,12 @@ version (CRuntime_DigitalMars)
     // workaround issue https://issues.dlang.org/show_bug.cgi?id=13727
     auto parallel(R)(R range, size_t workUnitSize) { return range; }
 }
+
+/** Wrapper around std.file.copy that also updates the target timestamp. */
+void copyAndTouch(RF, RT)(RF from, RT to)
+{
+    std.file.copy(from, to);
+    version (linux) {
```

Comment:
This functions behaviour should be consistent across all plattforms, maybe add a fallback lile:

```
import std.datetime.systime: Clock;
const time = Clock.currTime;
file.setTimes(time, time);
```

Reply:
Yes, I thought the windows implementation might update the timestamp but I confirmed it doesn't.  Thanks for the copy/paste implementation :)